src/styles/components/_checkbox.scss

```diff
@@ -0,0 +1,30 @@
+@import '../settings/colors';
+
+.w-checkbox__label {
```

Comment:
I'm not sure if this class is quite what we want, it seems too specific to checkbox and this one use case. Something more generic and reusable might be better.

If we want a generic class that makes all labels `inline-flex` then we should add it to `label` in the `generic/*` directory. But i'm not sure if we want that either...

Another option is to create some flexbox utility classes in the `overrides/*` directory. That may be a better approach.